### DISCUSSION TITLE
Statamic renders index instead of show and also...

### DISCUSSION THREAD

**Discussion by goodBranchUC** [statamic/cms#6356] [Q&A] [answered]
Hi there. I need help.
new to dev world.

hierarchy
projects/show...
projects/index...

my projects/show.antlers.html is rendering only the data in index.antlers.html

and projects/index.antlers.html only renders data from the CMS window via the 
Pages -> Projects -> Content textbox. Not from the index.antlers.html file.

And its rendering that content without requiring {{ content }} within the index.antlers.html

stumped.

**Comment by joshuablum** [comment, marked as answer]
Hey @goodBranchUC,

What's your collection's and entry's template set to? You can set the template for each of them. Either from the Control Panel or manually in the YAML of the collection or YAML front-matter of the entry's markdown file. Maybe you got it mixed up?

![CleanShot 2022-07-20 at 2 14 09](https://user-images.githubusercontent.com/3824203/179979373-7db1845d-cde5-424b-a0cf-ab2d4281d6c2.png)
![CleanShot 2022-07-20 at 2 15 54](https://user-images.githubusercontent.com/3824203/179979578-3e5a2da2-f15b-4dbe-bdae-7e52f1173563.png)
![image](https://user-images.githubusercontent.com/3824203/179979386-acf9e210-05ee-4855-b3fd-ea786bd59474.png)

**Reply by goodBranchUC** [reply]
![Screenshot_19](https://user-images.githubusercontent.com/99146550/180047325-69a677ff-a29a-4f9a-9a02-febaf7bf6952.png)

projects collection 

i dont see template for entry's.

**Reply by goodBranchUC** [reply]
i changed the projects collection's template to projects/show. which fixed that issue but im still not able to render anything from projects/index.antlers.html only from the content field. but i want to use html/tailwind.

**Reply by goodBranchUC** [reply]
how come my entry page doesn't have Template

![Screenshot_20](https://user-images.githubusercontent.com/99146550/180099217-02adfb70-03b3-4168-8552-d2fcfb71653c.png)


but in Jacks tutorial he does

![Screenshot_21](https://user-images.githubusercontent.com/99146550/180099383-5544875b-3873-4b72-b0d5-b9866d2e1f5c.png)
.

**Reply by joshuablum** [reply]
You have to explicitly add the `template` field to your collection's Blueprint ✌️
![CleanShot 2022-07-21 at 6 32 44@2x](https://user-images.githubusercontent.com/3824203/180266759-e949538c-fa3c-411f-ba32-be0488c904c4.png)

**Reply by joshuablum** [reply]
You could also have the `projects/index` be an entry of the `Pages` collection and the `projects/show` the default of the `Projects` collection. That's how I usually do it.

**Reply by goodBranchUC** [reply]
your initial answer worked. thanks.

**Reply by goodBranchUC** [reply]
I made an error. My Yamlfile wasn't the issue my projects.md file was. it generated
 template_field: projects/index instead of the working one
template: projects/index

**Comment by goodBranchUC** [comment]
For some odd reason my md file generated template_field: projects/index instead of template: projects/index.

had to delete _field and it worked.

**Reply by jasonvarga** [reply]
Sounds like you added a template field to your blueprint and didn't change the name. When you add a field it's always "fieldtype_field" until you change it.

**Reply by brunoalfred** [reply]
This thread is so helpful.
